src/nmodl/nocpout.cpp

```diff
@@ -735,7 +762,7 @@ extern Memb_func* memb_func;\n\
     if (for_netcons_) {
         Sprintf(buf, "\n#define _fnc_index %d\n", ppvar_cnt);
         Lappendstr(defs_list, buf);
-        ppvar_semantics(ppvar_cnt, "fornetcon");
+        ppvar_semantics(ppvar_cnt, "fornetcon", "_fnc_index", "void*" /* I think this is how it's currently used to avoid exposing an internal type */);
```

Comment:
`note`: something needs to be clarified before finalisation?

Reply:
Changed in 1e4916d9e.